readme.md

```diff
@@ -208,11 +229,18 @@ $service->loadIdentifier('Authentication.Orm', [
 ]);
 
 // Load the authenticators
-$service->loadAuthenticator('Authentication.Session');
-$service->loadAuthenticator('Authentication.Form');
+$service->loadAuthenticator('Authentication.Session', [
+    'sessionKey' => 'Auth'
+]);
+$service->loadAuthenticator('Authentication.Form', [
+    'fields' => [
+        'username' => 'email',
+        'password' => 'password'
+    ]
+]);
 ```
 
-While this seems to be a little more to write, the benefit is a greater flexibility and better [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).
+While this seems to be a little more to write, the benefit is a greater flexibility and better [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns). Plus, with this separation, it gives you much greater flexibility in developing your own authorization layer.
```

Comment:
Besides this being the third time mentioning authorization (I would remove it here as well), remove at least the `Plus,`.